src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -60,6 +60,138 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
 Refer to [Program.cs](../../examples/runtime-instrumentation/Program.cs) for a
 complete demo.
 
+## Metrics
+
+### GC related metrics
+
+| Name                                            | Description                                                          | Units     | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-------------------------------------------------|----------------------------------------------------------------------|-----------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**gc.collections.count** | Number of times garbage collection has occurred since process start. | `{times}` | ObservableCounter | `Int64`    | gen              | gen0, gen1, gen2 |
+
+Note: Collecting a generation means collecting objects in that generation and all
+its younger generations. However, each dimension for this metrics doesn't include
+the collection counts for the lower generation.
+e.g. count for gen1 is `GC.CollectionCount(1) - GC.CollectionCount(0)`.
+
+The API used to retrieve the value is:
+
+* [GC.CollectionCount](https://docs.microsoft.com/dotnet/api/system.gc.collectioncount):
+  The number of times garbage collection has occurred for the specified generation
+of objects.
+
+#### Additional GC metrics only available when targeting .NET Core 3.1 or later
+
+| Name                                           | Description                                                                                                                                                                                             | Units   | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**gc.allocations.size** | Count of the bytes allocated on the managed GC heap since the process start. .NET objects are allocated from this heap. Object allocations from unmanaged languages such as C/C++ do not use this heap. | `bytes` | ObservableCounter | `Int64`    |                  |                  |
```

Comment:
```suggestion
| process.runtime.dotnet.**gc.allocations.size** | Count of bytes allocated on the managed GC heap since the process start. .NET objects are allocated from this heap. Object allocations from unmanaged languages such as C/C++ do not use this heap. | `bytes` | ObservableCounter | `Int64`    |                  |                  |
```